tutorials/eda_contents/Computing_Environment/Unix.md

```diff
@@ -0,0 +1,363 @@
+# Intro to Unix
+
+An introduction to basic Unix commands and the file system.
+
+*The following notes are modications of both the book [An Introduction to Earth and Environmental Data Science](https://earth-env-data-science.github.io/intro.html) by Ryan Abernathey and the [Unix Shell tutorial ](http://swcarpentry.github.io/shell-novice/) that is freely available on the Software Carpentry website. It is recommended that readers take a look at the original works for further readings. The material is being used here under the terms of the [Creative Commons Attribution license](https://creativecommons.org/licenses/by/4.0/).*
+
+---
+
+## File and Directory Navigation
+
+In a unix environment, everything is considered a file.
+
+Getting started, select the **terminal** option from the **JupyterLab Launcher**. When loaded, you should see
+something like this.
+
+~~~
```

Comment:
Here and everywhere use the myst code syntax: 

https://jupyterbook.org/en/stable/reference/cheatsheet.html#code-and-syntax-highlighting

Reply:
updated to back ticks